monitoring/settings.py

```diff
@@ -11,10 +11,15 @@
 """
 
 import os
+import configparser
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+# Read configuration from the file
+cp = configparser.ConfigParser()
+file_path = os.path.join(BASE_DIR, 'conf', 'common.cfg')
+cp.read(file_path)
```

Comment:
Do we need some error handling in case the file doesn't exist or isn't parsable? (In case Aquilon messes something up?)

Reply:
Hmm! Having the error handling might be better than to NOT have one, Adrian. Updated the code to address the proposal. d50a671486dcf92392f0e84dbe1874abf75bdc71

Reply:
d50a671 doesn't look like it handles the case where the file is missing. Unless I am being dense...